app/controllers/hours_controller.rb

```diff
@@ -0,0 +1,68 @@
+class HoursController < ApplicationController
+  include MakeNotice
+
+  before_action :set_hour, only: [:show, :edit, :update, :destroy]
+  before_action :set_houred
```

Comment:
Not sure I like the word "houred" (verbing weirds language), maybe call it `volunteer` like in the model?